### PR TITLE
Add CO2STOR keyword in Runspec section

### PR DESCRIPTION
**Pull request by totto82**
The flag is used to trigger a CO2-BRINE specific PVT model.

### CODE REVIEW

**Comment by akva2** [comment]
you need to update the serialization. (Runspec::serializeOp(..))

File: src/opm/parser/eclipse/EclipseState/Runspec.cpp [hunk 1]
```diff
@@ -328,6 +329,11 @@ Runspec::Runspec( const Deck& deck ) :
                 OpmLog::note(msg);
             }
         }
+        if (runspecSection.hasKeyword("CO2STOR")) {
```
Comment (joakim-hove) [inline src/opm/parser/eclipse/EclipseState/Runspec.cpp:332]: Cool if you use the compiled constants - that particularly makes sense for the more volatile opm specific keywords:
```C++
#include <opm/parser/eclipse/ParserParserKeywords/C.hpp>

....
if (runspsecSection.hasKeyword<ParserKeywords::CO2STOR>()) {
    ...
    ...
}
```

**Review by joakim-hove — APPROVED** [review]

**Comment by totto82** [comment]
> you need to update the serialization. (Runspec::serializeOp(..))

Did I do it correctly? I also noticed that NUPCOL is missing.

**Comment by akva2** [comment]
yes, that's correct. nupcol should likely also be added, though i don't know where that is used downstream.

**Comment by akva2** [comment]
you should also add the new var to Runspec::serializeObject()

**Comment by totto82** [comment]
> you should also add the new var to Runspec::serializeObject(). This is used for the downstream serialization test, and it's good to have a value other than the default there.

Like this?

**Comment by akva2** [comment]
yup.

**Comment by totto82** [comment]
jenkins build this please

**Comment by totto82** [comment]
Was I just unlucky?

**Comment by akva2** [comment]
yes. your build picked up a change in opm-simulators, but not the upstream one in common.

**Comment by akva2** [comment]
jenkins build this please

**Comment by totto82** [comment]
jenkins build this please

**Comment by totto82** [comment]
Can this be merged now?

**Comment by akva2** [comment]
no.

https://ci.opm-project.org/job/opm-common-PR-builder/2472/gcc/new/source.28d127fb-21d4-47dc-8f18-fb2fd35f780b/#434

File: src/opm/parser/eclipse/EclipseState/Runspec.cpp [hunk 2]
```diff
@@ -418,6 +431,7 @@ bool Runspec::operator==(const Runspec& data) const {
            this->actdims() == data.actdims() &&
            this->saturationFunctionControls() == data.saturationFunctionControls() &&
            this->m_nupcol == data.m_nupcol;
+           this->m_co2storage == data.m_co2storage;
```
Comment (akva2) [inline src/opm/parser/eclipse/EclipseState/Runspec.cpp:434]: still wrong. replace semicolon on line above with &&

**Comment by totto82** [comment]
jenkins build this please

**Pull request merged**